### PR TITLE
VACMS-7070 out-of-box claro form style fixes

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
Relates to #7070

- fixed media cards for images/pdf
- removed divider from bottom of form above buttons
- made the delete button a scary red button
- updated list item bullets
- updated tags display for FAQ content type

- Did not adjust spacing issues for things like editorial workflow, hours of operation. Waiting until design tokens are finalized and we know what they should look like.
- Did not fix character count presentation for same reason, would like typographical design tokens.
- Did not adjust the select field length... the long one is because there is a very long name in the dropdown list ("Dr. Cameron McKinley Department of Veterans Affairs Veterans Center") and the field expands to that width.

## Testing done
- tested media with images, pdfs, multiple images (grid wraps after 3 now, all cards readable) on screen sizes 360px -> 1400px
- checked delete button is red & divider gone from node edit forms
- checked paragraph list bullet looks normal

## Screenshots
**before:**
Screen Shot 2021-11-26 at 9 16 39 AM

**after:**
bullet is normal size
Screen Shot 2021-11-30 at 2 16 01 PM

**before:**
Screen Shot 2021-11-26 at 10 08 27 AM

**after:** 
no line, button is red
Screen Shot 2021-11-30 at 2 04 53 PM

**before:**
Screen Shot 2021-11-26 at 10 15 35 AM
Screen Shot 2021-11-26 at 10 29 02 AM

**after:**
card widened slightly, text given some space to breathe, pdf table fits in card 
Screen Shot 2021-11-30 at 2 04 44 PM
Screen Shot 2021-11-30 at 2 14 30 PM

**before:**
Screen Shot 2021-11-26 at 12 52 30 PM

**after:**
Screen Shot 2021-11-30 at 2 03 28 PM

## QA steps

As user _uid_ with _user_role_
- check Vet Center, VAMC System, FAQ, News Release types to see media updates & others look as expected

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/themes/custom/vagovclaro/assets/scss/_paragraphs.scss [hunk 1]
```diff
@@ -1,143 +1,17 @@
-/* Paragraphs css for the proofing experience. */
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/assets/scss/_paragraphs.scss:1]: noting all of this removed code was moved to the `proofing.scss` file, the only place it was imported. this frees up the `_paragraphs.scss` filename for more universal paragraph form styling.

File: docroot/themes/custom/vagovclaro/templates/node-edit-form.html.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{#
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/templates/node-edit-form.html.twig:1]: noting this is identical to claro's out-of-box `node-edit-form.html.twig`, minus a single div providing a divider between the node form buttons above `{{form.footer}}` at the end of page.

File: docroot/themes/custom/vagovclaro/assets/scss/_paragraphs.scss (hunk 1 above)
Comment (swirtSJW) [reply docroot/themes/custom/vagovclaro/assets/scss/_paragraphs.scss:1]: Thanks for clarifying.   Makes code review so much easier.

**Review by swirtSJW — APPROVED** [review]
Looks great.   Thank you so much for all the comparative screen shots.

**Pull request merged**